### PR TITLE
Collab3 notifications

### PR DESCRIPTION
**Pull request by justinchoi001**

### CODE REVIEW

File: docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Managing Notifications and Requests
+
+Users receive notifications and requests if they have subscribed to a blog or message board, or if someone sent them a private message, invitation, event reminder, or were mentioned in a post.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md]: This sentence is really long. Can we clean this up or break it into separate sentences?

File: docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    ![User Avatar and notifications](./managing-notifications-and-requests/images/01.png)
+
+1. The _Notifications List_ tab shows all unread notifications. Here, the user was mentioned in a Message Boards comment.
+
+    ![User Avatar and notifications](./managing-notifications-and-requests/images/02.png)
+
+## Managing Notifications
+
+Notifications can pile up after some time, especially if you were away for a few days. To filter and sort your notifications:
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md]: Simplify this:

`To manage and sort accumulated notifications:`

File: docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+1. Several icons appear:
+    * **Mark as Read** (![Open Envelope](./managing-notifications-and-requests/images/icon-envelope-open.png))
+    * **Mark as Unread** (![Closed Envelope](./managing-notifications-and-requests/images/icon-envelope-closed.png))
+    * **Delete** ![Delete Button](./managing-notifications-and-requests/images/icon-delete.png))
+
+    ![Managing Multiple Notification status](./managing-notifications-and-requests/images/05.png)
+
+## Managing Requests
+
+When a user gets a request to join a site, it appears in the *Requests List* tab. Users have the option to click _Confirm_ to join the site or click _Ignore_ to decline.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md]: Does this only apply to site memberships, or does it also apply to workflow as well? I assume that workflow requests would also show up in requests. Please double check.

File: docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/mentioning-users.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Mentioning Users
+
+Users can "mention" other users by entering the `@` character in front of each user's user name (`@john.smith`) in a blog entry, [a message boards post](../../message-boards/user-guide/creating-message-boards-threads.md), or comments in any app that supports comments.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/mentioning-users.md]: Can we add a sentence that describes the 'what' "@" mentioning another user does? I assume the sentence might be:

`Mentioning another user sends a (notification)[link] to that user.` 

Or something to that effect?

File: docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/mentioning-users.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Using the Mentions Function in a _Message Boards_ Thread
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/mentioning-users.md]: Italics not necessary for the header

File: docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
+# Managing Notifications and Requests
```
Comment (jrhoun) [inline docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md]: Should there be a new placeholder article for "Configuring the Notification Email Template"? I assume that there is an email that goes out for notifications for a user and that there is an email template that can be configured. If there is - let's link that and add a placeholder (Coming soon) for it.

File: docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/mentioning-users.md (hunk 5 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/mentioning-users.md]: removed italics

File: docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md (hunk 2 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md]: Revised.

File: docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md (hunk 1 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md]: Revised

File: docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/mentioning-users.md (hunk 4 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/mentioning-users.md]: Revised

File: docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md (hunk 3 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md]: The Requests in this context are for Sites. Workflows are integrated with notifications and workflow requests appear in the standard notifications tab. 

Revised.

File: docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md (hunk 6 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md]: I didn't see anything that was instance-wide for configuring email templates for notifications; at least on the front end. What you're thinking about may be Commerce only? The email notifications in Commerce were site specific.

File: docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md (hunk 3 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md]: The requests here are for site invitations.

File: docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md (hunk 6 above)
Comment (jrhoun) [reply docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md]: image

Comment (jrhoun) [reply docs/dxp/7.x/en/collaboration-and-social/notifications-and-requests/user-guide/managing-notifications-and-requests.md]: I think ^ is what I was thinking of - we should create a placeholder for an article that covers the above functionality - do this in another pull.

**Comment by jrhoun** [comment]
Merged. Thank you. See changes I added.

**Pull request closed without merging**